sabnzbd/nzbstuff.py

```diff
@@ -1172,6 +1173,8 @@ def remove_article(self, article: Article, success: bool):
                 # Check the availability of these first articles
                 if cfg.fail_hopeless_jobs() and cfg.fast_fail():
                     job_can_succeed = self.check_first_article_availability()
+                    if not job_can_succeed:
+                        abort_reason = "https://sabnzbd.org/not-complete (check_first_article_availability)"
```

Comment:
I understand you want to specify, but this is not usefull for regular users, let's not modify the text.
No need for `abort_reason`.